quickstarts/lighttpd/config.yml

```diff
@@ -0,0 +1,40 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring quickstart for Lighttpd
+
+  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure and adaptable.
+
+  ### Why monitor Lighttpd?
+
+  Lighttpd monitoring provides important information to help the developers construct a complete picture of a web server's performance. This data comprises uptime, network in bytes and packets, number of connections and more.
+
+  ### What’s included?
+
+  The Lighttpd monitoring quickstart from New Relic provides immediate full-stack observability:
+  - Alerts (average number of connections, average number of requests, high memory usage).
+  - Dashboards (uptime, connection state handle request, connection state still alive, networks, requests per second and more).
+  - Be proactive in monitoring potential issues and resolving them as soon as possible.
+
+summary: |
+   With our Infrastructure agent and Flex configuration setup, you can monitor and validate your Lighttpd processes. Right out of the box, analyze and capture performance data.
```

Comment:
```suggestion
   With our infrastructure agent and Flex configuration setup, you can monitor and validate your lighttpd processes. Right out of the box, analyze and capture your server's performance data.
```

Reply:
Flex is an application-agnostic, all-in-one [New Relic integration](https://docs.newrelic.com/docs/integrations) with which you can instrument any app that exposes metrics over a standard protocol (HTTP, file, shell) in a standard format.
 Sharing [github url](https://github.com/newrelic/nri-flex) for the better understanding.

Reply:
Thank you!